packages/eas-cli/src/prompts.ts

```diff
@@ -45,14 +45,16 @@ export async function confirmAsync(
 export async function selectAsync<T>(
   message: string,
   choices: ExpoChoice<T>[],
-  options?: Options
+  options?: Options,
+  warn?: string
```

Comment:
It's not obvious what warn here means, maybe `warningMessageForDisabledEntries` or some comment?